signature-aggregator/README.md

```diff
@@ -0,0 +1,64 @@
+# Signature Aggregator
+
+This directory contains a lightweight stand-alone API for requesting signatures for a Warp message from subnet validators.
+It is also used by the relayer for gathering signatures when configured to use AppRequest instead of the WarpSignature RPC client.
```

Comment:
```suggestion
It is also used by `awm-relayer` for gathering signatures when configured to use AppRequest instead of the WarpSignature RPC client.
```

Let's be specific